grafana/src/eaRelease.jsonnet

```diff
@@ -0,0 +1,276 @@
+// https://grafana.com/docs/grafana/latest/dashboards/json-model/
+local grafana = import 'grafonnet/grafana.libsonnet';
+local graphPanel = grafana.graphPanel;
+local timeseries = grafana.timeseries;
+local textPanel = grafana.text;
+local prometheus = grafana.prometheus;
+
+local shared = import './shared.libsonnet';
+local addSideLegend = shared.helpers.addSideLegend;
+
+local cortexDataSource = shared.constants.cortexDataSource;
+local instanceFilter = shared.constants.instanceFilter;
+local interval = shared.constants.interval;
+
+local templates = shared.createTemplates(multiService=true);
+
+/**
+ * Panels
+ */
+local inboundRequestsNodeEAPanel = textPanel.new(
+    content='# Inbound Requests (Node → EA)',
+);
+
+local ErrorRate5xxPanel = graphPanel.new(
+  title='Top 10 - 5xx Error Rate',
+  datasource=cortexDataSource,
+  format='percent',
+).addTarget(
+  prometheus.target(
+    'topk(10, sum by(app_name) (rate(http_requests_total{' + instanceFilter + ',status_code=~"^5..",app_name=~"$adapter.*"}' + interval + ')) / sum by(app_name) (rate(http_requests_total{app_name!="",' + instanceFilter + ',app_name=~"$adapter.*"}' + interval + ')) * 100)'
+  )
+);
+
+local ErrorRate4xxPanel = graphPanel.new(
+  title='Top 10 - 4xx Error Rate',
+  datasource=cortexDataSource,
+  format='percent',
+).addTarget(
+  prometheus.target(
+    'topk(10, sum by(app_name) (rate(http_requests_total{' + instanceFilter + ',status_code=~"^4..",app_name=~"$adapter.*"}' + interval + ')) / sum by(app_name) (rate(http_requests_total{app_name!="",' + instanceFilter + ',app_name=~"$adapter.*"}' + interval + ')) * 100)'
+  )
+);
+
+local Bot10CacheHitTotalReqCount = graphPanel.new(
+  title='Bot 10 - (Cache Hit / Total Req Count) Ratio',
+  datasource=cortexDataSource,
+  format='percent',
+).addTarget(
+  prometheus.target(
+    'bottomk(10, (sum(rate(http_requests_total{' + instanceFilter + ' ,type=\"cacheHit\",app_name=~"$adapter.*"}' + interval + ')) by (app_name) / sum(rate(http_requests_total{' + instanceFilter + ' ,app_name=~"$adapter.*"}' + interval + ')) by (app_name))) * 100'
```

Comment:
This should filter by service instead of app_name, to differentiate between sub-adapters (i.e. adapter-x, adapter-x-solana)

Reply:
I don't see a filter field in http_requests_total, should I use the instance field instead?